.github/workflows/version-packages.yml

```diff
@@ -39,6 +40,6 @@ jobs:
         if: ${{ steps.has-changesets.outputs.has-changesets == 'true' }}
         uses: changesets/action@b98cec97583b917ff1dc6179dd4d230d3e439894
         with:
-          version: yarn changeset version && yarn generateVersions
```

Comment:
`&&` in this script wasn't getting parsed correctly:

```
$ /home/runner/work/amplify-ui/amplify-ui/node_modules/.bin/changeset version '&&' yarn generateVersions
🦋  error Too many arguments passed to changesets - we only accept the command name as an argument
```
consolidating the two scripts into its own script to fix it.

Reply:
interesting. this seems cleaner anway, but I wonder if this command needed to be in quotes.